### PR TITLE
Fix: Certain tests were failing due to MFC values, which is handeled

### PR DESCRIPTION
**Pull request by vrbagalkote**
Tests such as ghs_dhs rebuild online_offline were failing, due to perticular
MFC values, and all those scenario are handeled now. Also changes made for migrate
test case, if there is no enough drives to pop for migration, test will exit.

Signed-off-by: Venkat R B <vrbagal1@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by vrbagalkote** [comment]
Attached is job.log file
[job.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/1534434/job.txt)

File: io/disk/Avago_storage_adapter/avago9361_vd.py [hunk 1]
```diff
@@ -156,6 +163,8 @@ def test_online_offline(self):
                 self.set_online_offline(state)
                 time.sleep(10)
         self.vd_delete()
+        cmd = "%s /c%d set autorebuild=on" % (self.tool, self.controller)
+        self.check_pass(cmd, "Failed to set auto rebuild on")
```
Comment (narasimhan-v) [inline io/disk/Avago_storage_adapter/avago9361_vd.py]: Guess this is not needed, as they are already handled in `setUp()` and `tearDown()`

**Comment by narasimhan-v** [comment]
@vrbagalkote There are more changes done. Please explain all in the commit message.

**Comment by PraveenPenguin** [comment]
@narasimhan-v thanks for review

**Pull request merged**